src/test/java/uk/gov/hmcts/reform/sscs/services/ccd/UpdateCcdRepresentativeTest.java

```diff
@@ -137,13 +149,19 @@ public void givenARepContactChangeWhenExistingContactIsNull_willChangeDataAndRet
                 Representative.builder().name(Name.builder().lastName("Potter").build())
                     .contact(null).build()
             ).build())
+                .subscriptions(Subscriptions.builder().representativeSubscription(
+                        Subscription.builder().tya(ABCDEFGH_2).email("harry.potter@wizards.com").subscribeEmail(NO)
+                                .subscribeSms(NO).build()).build())
             .build();
 
         boolean hasDataChanged = UpdateCcdRepresentative.updateCcdRepresentative(gapsCaseData, existingCaseData);
         assertTrue("rep contact has changed", hasDataChanged);
-        assertEquals(gapsCaseData.getAppeal().getRep(), existingCaseData.getAppeal().getRep());
-        assertEquals(gapsCaseData.getSubscriptions().getRepresentativeSubscription(),
-            existingCaseData.getSubscriptions().getRepresentativeSubscription());
+        // should not change the existing tya number even though we have new tya number on gapsCaseData
```

Comment:
could you remove the comment and move it into a method with a meaningful name, please.